src/pages/Address/AddressAssetComp.tsx

```diff
@@ -61,7 +65,15 @@ export const AddressSudtComp = ({ account }: { account: SUDT }) => {
   )
 }
 
-export const AddressSporeComp = ({ account }: { account: Spore }) => {
+export const AddressSporeComp = ({
+  account,
+  isRGBPP,
+  udtLabel,
```

Comment:
It's `spore comp` so the label should be `spore`

Reply:
done